plugins/discourse-narrative-bot/plugin.rb

```diff
@@ -300,4 +300,17 @@ def fetch_avatar(user)
     DiscourseNarrativeBot::BOT_USER_ID,
     "discobot@discourse.org"
   )
+
+  PostGuardian.class_eval do
+    alias_method :existing_can_create_post?, :can_create_post?
+
+    def can_create_post?(parent)
```

Comment:
Just curious, why do we need to monkey patch the original method?

Reply:
If `enable_system_message_replies` is false and `discourse_narrative_bot_enabled` is true it seems to be possible to get in to a state where you can't reply to the Discobot messages. So the following three lines essentially always allow replies to Discobot.

Reply:
I think we should avoid the monkey patch for now. If an admin is going to disable PM and not enable `SiteSetting.enable_system_message_replies`, it is them intentionally breaking discobot. Maybe if that is the case, discobot shouldn't even send out the initial message.